### PR TITLE
Added Pure_Parallels folder with update_parallel_wcs.py script

### PR DESCRIPTION
**Pull request by goudfroo**
Added Pure_Parallels folder contain a script to help users correct the WCS in pure parallel exposures (which are currently off by of order 10-100 mas). This is a workaround for users while Operations and I&T addresses the issue (which may well take several months to get implemented). There is also a README.md file to inform users on how to use the script.

### CODE REVIEW

File: Pure_Parallels/update_parallel_wcs.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+import glob
+from astropy.io import ascii
+import astropy.io.fits as pyfits
+import numpy as np
+
+def log_comment(LOGFILE, comment, verbose=False, show_date=False, mode='a'):
+    """
+    Log a message to a file, optionally including a date tag
+    """
+    import time
```
Comment (tbainesUA) [inline Pure_Parallels/update_parallel_wcs.py:12]: To adhere to the PEP8 standards all imports should be at the top of the file.

File: Pure_Parallels/update_parallel_wcs.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    verbose : bool
+        Status messaging
+    
+    Returns
+    -------
+    status : None, True
+        Returns None if some problem is found 
+    
+    """
+    import pysiaf
```
Comment (tbainesUA) [inline Pure_Parallels/update_parallel_wcs.py:62]: Again imports should be located at top of file (PEP8 - standards).

File: Pure_Parallels/update_parallel_wcs.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
+import os, sys
```
Comment (tbainesUA) [inline Pure_Parallels/update_parallel_wcs.py:1]: imports should be on separate lines specified by PEP8 standards. 

import os
import sys

**Review by tbainesUA — CHANGES_REQUESTED** [review]
Just need to fix a few things to be compliant with PEP8 standards.

File: Pure_Parallels/update_parallel_wcs.py (hunk 1 above)
Comment (goudfroo) [reply Pure_Parallels/update_parallel_wcs.py]: done

File: Pure_Parallels/update_parallel_wcs.py (hunk 2 above)
Comment (goudfroo) [reply Pure_Parallels/update_parallel_wcs.py]: done

File: Pure_Parallels/update_parallel_wcs.py (hunk 3 above)
Comment (goudfroo) [reply Pure_Parallels/update_parallel_wcs.py]: done

**Comment by goudfroo** [comment]
Made those requested changes, committed them and pushed them to the same branch.

**Review by tbainesUA — APPROVED** [review]
Looks goo, no other action needed.

**Pull request merged**